docs/entity-editor.rst

```diff
@@ -0,0 +1,63 @@
+######################
+The Entity Editor
+######################
+
+Introduction
+==============
+
+In Bookbrainz, a set of information about a particular author, book, publication, magazine, etc. is referred to as an *Entity*, and the form which we use to add/edit this information is called the **Entity-Editor**.
+We currently have the following types of entities in Bookbrainz:
+
+* Author
+* Work
+* Edition
+* Edition-Group
+* Series 
+* Publisher
+
```

Comment:
It would be good here to give a broad overview of the editor without going into details:
* the code for the editor is in its own folder in the codebase
* we use Redux for state management
* validation code is also in folder
* there are separate components (but that share common code) for editing entities and for merging entities
* Anything else?

Reply:
That seems to cover it all. Have added this.